### DISCUSSION TITLE
Headers of the inner dependency are not found

### DISCUSSION THREAD

**Discussion by wellbranding** [callstack/react-native-builder-bob#404] [Q&A]
I am developing a native module for our SDK. I had zero issues with the Android setup, but keep getting issues with the IOS project. The structure of our project is the following:
1. The root IOS folder creates a cocoa pods project and includes our native SDK with the following Podfile setup:
target 'IdenfyReactNative' do
  use_frameworks!
  pod 'iDenfySDK'
end
2. The example project includes this native module dependency via Podfile:
pod 'idenfy-react-native', :path => '../..'

When I try to build the project I am getting the following error from one of the inner dependencies, **defined in the iDenfySDK podspec**:

**'MotionInterchange/MotionInterchange.h' file not found**
<img width="1393" alt="Screenshot 2020-10-18 at 22 16 50" src="https://user-images.githubusercontent.com/31806066/96377649-a15aa580-118f-11eb-89bc-679315bfe21e.png">

This approach works perfectly on native IOS development. However, I can't solve these issues in ReactNative.

What I did found strange is the Development Pods folder. Our idenfy-react-native is located in this folder and it also contains all inner frameworks and **their private headers**, which are the ones who cause the built failure in the first place(they are not visible if you include our iDenfySDK into regular IOS project)

Perhaps, there is a different way of including the native module, which internally **has inner dependencies** as well?

**Comment by adrianpandelea-sv** [comment]
Hello, I have the same issue. Did you manage to find a fix for it? Thanks

**Comment by murtazabeans** [comment]
I am also facing the same issue. Has anyone got any workaround for this?

**Comment by TfADrama** [comment]
Have you tried to just put your pod dependency on your bob lib podspec and delete that pod dependency that you added on the ios pod file?
```
  s.dependency "React" --> This is a pod
  s.dependency "ReachabilitySwift" --> This is another pod
  s.vendored_frameworks = 'PSReader.xcframework', 'ZIPFoundation.xcframework' --> These are .frameworks or .xcframeworks
```
